config/initializers/admin_constants.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+SUPER_ADMIN = 'super_admin'
+ADMIN       = 'admin'
+ROW_USER    = 'user'
+
+PROHIBITED_ACCESS             = 'You have no permitions to do this, please log in as a super_admin'
```

Comment:
i am not sure if this is correct way to do this. It probably would be better to just return to notice with just strings not constants.
CC @mmrshk